src/PhpSpec/Formatter/FatalFormatter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Formatter;
+
+use PhpSpec\Console\IO;
+use PhpSpec\Formatter\Presenter\PresenterInterface;
+use PhpSpec\Listener\StatisticsCollector;
+use PhpSpec\Message\Example;
+use PhpSpec\Message\MessageInterface;
+
+class FatalFormatter extends ConsoleFormatter
```

Comment:
This is not a formatter - formatters are selected with the `--formatter` CLI option and handle the output of the whole test execution.

You're not really using any features of ConsoleFormatter so you should drop the dependency and rename this class.